### DISCUSSION TITLE
Filter by tag

### DISCUSSION THREAD

**Discussion by TefDesign** [dailydotdev/daily#999] [Feature Requests]
Hi ! 
Is it possible to add this feature : filter by tag in feed. Example, my feed contain articles for Frontend but i want show only article for CSS. Currently, I don't think it's possible, or I haven't worked out how to do it., 

Thanks !

**Comment by idoshamun** [comment]
Click the "Feed Settings" button on the main page and from there you can choose the tags you like.
<img width="236" alt="image" src="https://github.com/dailydotdev/daily/assets/1993245/4a310d60-9aaf-407e-b07c-1f27da551055">

**Reply by TefDesign** [reply]
Sorry @idoshamun , I don't think you understood what I meant: Feed settings lets you configure what you want in your feed. I'm talking about being able to filter within your feed. If I've got 20 topics selected, I'm not going to deactivate everything just to look at a language temporarily and then check everything again. :)

**Reply by idoshamun** [reply]
Oh sorry! You can use this tag feed, it follows this url structure: `https://app.daily.dev/tags/css`.
Also if you click on any tag it will take you there

**Reply by TefDesign** [reply]
HO !!! Yes thanks !!!! Excellent :)
